venvs/converge.py

```diff
@@ -68,9 +67,9 @@ def main(filesystem, locator, link_dir, handle_error):
                 if pytoml.loads(existing_config.read()) == config:
                     continue
         except FileNotFound:
-            virtualenv.create(arguments=arguments)
+            virtualenv.create(python=python)
```

Comment:
There are tests missing which make it easy to miss, but I think you've dropped passing along arguments here now.

Reply:
Arguments was set above as `arguments = ["-p", python]` and doesn't otherwise exist.

Reply:
Ah, this is converge... OK, what that means in practice is that it's not currently possible to pass along options to virtualenv from here, but I've started to think that's a bad idea anyhow, so I guess never mind my comment.